webpack.config.js

```diff
@@ -29,6 +29,11 @@ const WORDPRESS_NAMESPACE = '@wordpress/';
 
 const gutenbergPackages = Object.keys( dependencies )
 	.filter( ( packageName ) => packageName.startsWith( WORDPRESS_NAMESPACE ) )
+	// TODO fix config to build them as well.
+	.filter( ( packageName ) =>
+		! ( packageName.includes( 'react-native-editor' ) ) &&
```

Comment:
Could simply use `packageName.startsWith( 'react-native' )`.
Btw do you think we can have webpack build our native bundles? That would be awesome :)

Reply:
>Could simply use packageName.startsWith( 'react-native' ).

Yeah, good idea :)

>Btw do you think we can have webpack build our native bundles? That would be awesome :)

This is what Haul does, so it is possible, but TBH i'm not sure if it should be a part of this PR :) WDYT?

Reply:
> This is what Haul does, so it is possible

Awesome :)

> but TBH i'm not sure if it should be a part of this PR :) WDYT?

Yeah probably not.
If we could have minimal changes in this PR that would be great